dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -162,30 +235,45 @@ export const Columns: React.FC<{
 		)}
 
 		<li>
-			<ThemeProvider theme={{ ...buttonThemeBrand }}>
-				<div css={searchBar}>
-					<LinkButton
-						href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-						tabIndex={-1}
-						className="selectableMenuItem"
-						priority="secondary"
-						icon={
-							<SvgMagnifyingGlass
-								isAnnouncedByScreenReader={true}
-								size="medium"
-							/>
-						}
-						aria-label="Search with google"
-						data-link-name="nav2 : search : submit"
-						type="submit"
-					>
-						Search
-					</LinkButton>
-				</div>
-			</ThemeProvider>
+			<form css={searchBar} action="https://www.google.co.uk/search">
+				<TextInput
+					hideLabel={true}
+					label="Search input"
+					cssOverrides={searchInput}
+					name="q"
+					placeholder="Search"
+					data-link-name="nav2 : search"
+				/>
 
+				<Label hideLabel={true} text="google-search">
+					<div css={searchGlass}>
+						<SvgMagnifyingGlass
+							isAnnouncedByScreenReader={true}
+							size="medium"
+						/>
+					</div>
+				</Label>
+				<Button
+					icon={
+						<SvgArrowRightStraight
+							isAnnouncedByScreenReader={true}
```

Comment:
I'd be interested as to how [this text](https://github.com/guardian/source/blob/529f25a8321ec14a51bb0045604c81a48102e1a0/packages/%40guardian/source-react-components/vendor/icons/SvgArrowRightStraight.tsx#L37) being read out is useful?